pkg/logs/tailers/socket/tailer.go

```diff
@@ -71,7 +71,9 @@ func (t *Tailer) forwardMessages() {
 		if len(output.GetContent()) > 0 {
 			origin := message.NewOrigin(t.source)
 			origin.SetTags(output.ParsingExtra.Tags)
-			t.outputChan <- message.NewMessage(output.GetContent(), origin, output.Status, output.IngestionTimestamp)
+			// Preserve ParsingExtra information from decoder output (including IsTruncated flag)
+			msg := message.NewMessageWithParsingExtra(output.GetContent(), origin, output.Status, output.IngestionTimestamp, output.ParsingExtra)
+			t.outputChan <- msg
```

Comment:
And here, could we just use `output` rather than creating a new message. We seem to do that in the `windowsevent` tailer (just reuse the decoded message).